internal/auth/options.go

```diff
@@ -57,65 +59,65 @@ import (
 // StatsdPort - port where statsd client listens
 // StatsdHost - host where statsd client listens
 type Options struct {
-	RedirectURL       string `envconfig:"REDIRECT_URL" `
-	ClientID          string `envconfig:"CLIENT_ID"`
-	ClientSecret      string `envconfig:"CLIENT_SECRET"`
-	ProxyClientID     string `envconfig:"PROXY_CLIENT_ID"`
-	ProxyClientSecret string `envconfig:"PROXY_CLIENT_SECRET"`
+	RedirectURL       string `mapstructure:"redirect_url" `
```

Comment:
Do we need these `mapstructure` to support the `_` from the env vars?

Reply:
As I understand it, internally viper stores the env vars in a map and then uses https://github.com/mitchellh/mapstructure to decode into the struct, so the tags here need to be `mapstructure`, instead of `envconfig`